.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+Please describe the motivation for this PR and provide enough
+information so that others can review it.
+
+Please mark with X for applicable areas.
+
+- [ ] New Functionality
+- [ ] Documentation
+- [ ] Install
+- [ ] Control Plane
+- [ ] CLI Tool
+- [ ] Certificate Management
+- [ ] Networking
+- [ ] Metrics
+- [ ] Security
+- [ ] Tests / CI System
+- [ ] Other
+
+Please answer the following questions with yes/no.
+
+- Does this change contain code from or inspired by another project? If so, did you notify the maintainers and provide attribution?
```

Comment:
Looks like we lost the yes/no

Reply:
Didn't follow, what do you suggest we add?

Reply:
The `yes/no` checkbox was dropped intentionally to make users answer the question explicitly. Having a checkbox can lead to accidental marking of checkbox which could lead to serious problems, so making users answer with `yes/no` seems like a good choice.